chassis/invokers/src/main/java/com/griddynamics/jagger/invoker/http/v2/JSpringBasedHttpClient.java

```diff
@@ -0,0 +1,52 @@
+package com.griddynamics.jagger.invoker.http.v2;
+
+import org.springframework.http.RequestEntity;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.client.RestTemplate;
+
+import java.net.URI;
+import java.util.Map;
+
+/**
+ * Implementation of {@link JHttpClient}. <p>
+ * This implementation is based on the Spring {@link RestTemplate}.
+ *
+ * @author Anton Antonenko
+ * @see JHttpClient
+ * @since 1.2.7
+ */
+@SuppressWarnings({"unused", "unchecked"})
+public class JSpringBasedHttpClient implements JHttpClient {
```

Comment:
Well, I would call it something like "DefaultJHttpClient", i.e. w/o specifying implementation details.